src/pages/projects/index.page.tsx

```diff
@@ -86,6 +94,17 @@ function Projects() {
     return _pv.length ? _pv : [PV_V1, PV_V2]
   }, [includeV1, includeV2])
 
+  function updateRoute(
```

Comment:
added this helper to unify updating the query params, which is done when either searchText or tags are changed